nextcloudappstore/core/static/assets/css/fonts.css

```diff
@@ -0,0 +1,120 @@
+/* Generated by Font Squirrel (https://www.fontsquirrel.com) on July 12, 2016 */
+
+
+
+@font-face {
+    font-family: 'open_sanssemibold_italic';
```

Comment:
You can just copy the fonts.css file from the server: https://github.com/nextcloud/server/blob/master/core/css/fonts.css